cookbook/bundles/best_practices.rst

```diff
@@ -221,7 +218,7 @@ following standardized instructions in your ``README.md`` file.
     following command to download the latest stable version of this bundle:
 
     ```bash
-    $ composer require <package-name> "~1"
+    $ composer require <package-name>
```

Comment:
I would leave the version number. If you don't add it, you will very likely get into support trouble one you release a new major version and people complain about BC breaks because of reading old documentations.